src/Microsoft.ML.Transforms/doc.xml

```diff
@@ -57,6 +57,61 @@
       </example>
     </example>
 
+    <member name="CountFeatureSelection">
+      <summary>
+        Selects the slots for which the count of non-default values is greater than or equal to a threshold.
+      </summary>
+      <remarks>
+        <para>
+          This transform uses a set of aggregators to count the number of non-default values for each slot and
+          instantiates a <see cref="T:Microsoft.ML.Transforms.SlotsDroppingTransformer"/> to actually drop the slots.
+          This transform is useful when applied together with a <see cref="T:Microsoft.ML.Transforms.OneHotHashEncodingTransformer"/>. 
+          The count feature selection can remove those features generated by the hash transform that have no data in the examples.
+        </para>
+      </remarks>
+    </member>
+    <example name="CountFeatureSelection">
+       <example>
+        <code language="csharp">
+          pipeline.Add(new FeatureSelectorByCount
+          { 
+            Column = new[]{ &quot;Feature1&quot; }, 
+            Count = 2 
+          });
+        </code>
+      </example>
+    </example>
+
+    <member name="MutualInformationFeatureSelection">
+      <summary>
+        Selects the top k slots across all specified columns ordered by their mutual information with the label column.
+      </summary>
+      <remarks>
+        <para>
+          The mutual information of two random variables X and Y is a measure of the mutual dependence between the variables.
+          Formally, the mutual information can be written as:
+        </para>
+        <para>I(X;Y) = E[log(p(x,y)) - log(p(x)) - log(p(y))]</para>
```

Comment:
Wasn't there a latex formula that Wei-Sheng has created just recently for this?

Reply:
This isn't applicable to the categorical transforms